4/Dockerfile

```diff
@@ -20,10 +18,10 @@ ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk \
 WORKDIR /opt
 
 RUN set -eux; \
-    addgroup -S -g ${GID} scanner-cli; \
-    adduser -S -D -u ${UID} -G scanner-cli scanner-cli; \
-    apk add --no-cache --virtual build-dependencies wget unzip gnupg; \
-    apk add --no-cache git python3 py-pip bash shellcheck 'nodejs>12' openjdk11-jre curl musl-locales musl-locales-lang; \
+    groupadd --system --gid 1000 scanner-cli; \
+    useradd --system --uid 1000 --gid scanner-cli scanner-cli; \
+    apt-get update; \
+    apt-get install -y --no-install-recommends gnupg unzip wget bash fonts-dejavu python3 python3-pip shellcheck nodejs>12 build-essential; \
```

Comment:
@jCOTINEAU  `nodejs>12` is not valid for apt-get. This will probably just install the jammy version of nodejs which is 12.22.9 and is incompatible as SonarCloud will error out:  "The version of node.js (12) you have used to run this analysis is deprecated and we stopped accepting it."

https://github.com/SonarSource/sonar-scanner-cli-docker/actions/runs/4764684758/jobs/8469570881#step:7:77